src/app/notes/notes.service.ts

```diff
@@ -35,15 +41,30 @@ export class NotesService {
    *
    * @returns The Note list
    */
-  toNoteList(jsonArray: any[]): Note[] {
+  toNoteList(res: any[]): Note[] {
     const list = [];
 
-    for (let i = 0, len = jsonArray.length; i < len; i++) {
-      for (const value of Object.values(jsonArray[i])) {
+    for (let i = 0, len = res.length; i < len; i++) {
+      for (const x of Object.values(res[i][0])) {
+        const value: any = x;
+
+        // Set the release version from the asset path
+        value.release_version = res[i][1];
+
+        // Add the converted value to the results
         list.push(value);
       }
     }
 
     return list;
   }
+
+  /**
+   * Retrieve the release version from a asset path
+   *
+   * @returns The release version
+   */
+  private releaseVersionFromPath(path: string): string {
+    return path.replace(/^.*release-notes-/, '').replace(/.json$/, '');
```

Comment:
:eye_speech_bubble: I think we should note in a manual somewhere that filenames now _MUST_  (in 
 an RFC way) conform to this pattern.

Reply:
The tooling ensures that, right? :)